test/models/explainer_test.rb

```diff
@@ -0,0 +1,100 @@
+require_relative '../test_helper'
+
+class ExplainerTest < ActiveSupport::TestCase
+  def setup
+    super
```

Comment:
Please try to run the tests here without `super`, since `super` makes everything slower. In latest models, we've been trying to not use it and then just add the setup steps we actually need.